app/scripts/lib/app-start.js

```diff
@@ -641,27 +641,6 @@ define(function (require, exports, module) {
       }
     },
 
-    _getErrorPage: function (err) {
-      if (AuthErrors.is(err, 'INVALID_PARAMETER') ||
```

Comment:
This list always bothered me. Having the logic in app-start also meant it couldn't be reused elsewhere.